jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -61,3 +61,18 @@ set -e
 
 docker-compose -f docker-compose-mini.yml up > $WORKSPACE/build-log/vagrant.log &
 popd
+
+mountpath="$WORKSPACE/RackHD/docker/files/mount"
```

Comment:
as @kellylu2sym  found, there's RackHD/docker/files/mount/common folder in docker compose file to be checked.